activejob/lib/active_job/exceptions.rb

```diff
@@ -156,7 +156,8 @@ def determine_delay(seconds_or_duration_or_algorithm:, executions:, jitter: JITT
         jitter = jitter == JITTER_DEFAULT ? self.class.retry_jitter : (jitter || 0.0)
 
         case seconds_or_duration_or_algorithm
-        when :exponentially_longer
+        when :exponentially_longer, :with_polynomial_backoff
```

Comment:
We should deprecate the old name

Reply:
Yep! Done in https://github.com/rails/rails/pull/49292/commits/9246a9f9172f750375c1285a52d1175e4bd2b00c
Testing for the deprecation made me realized that I was using the wrong exception name in that test 🙈